README.md

```diff
@@ -102,6 +102,18 @@ steps:
 - `build_environment_variables_file`: (Optional) Path to a local YAML file
   containing variables. See 'env_vars_file' for syntax.
 
+- `docker_repository`: (Optional) User managed repository created in Artifact
```

Comment:
Lets clarify expected input is of form `projects/${PROJECT}/locations/${LOCATION}/repositories/${REPOSITORY}`

Reply:
Also looks like cross project image repos as are not allowed. Maybe we can just link to https://cloud.google.com/sdk/gcloud/reference/beta/functions/deploy#--docker-repository

Reply:
Done